setup/ci/gcb.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: single
+title:  "Google Cloud Build"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Setting up [Google Cloud Build](https://cloud.google.com/cloud-build/) as a Continuous Integration (CI)
+system within Spinnaker enables triggering pipelines when builds complete.
+
+## Prerequisites
+
+You need to have a [Google Cloud Platform](https://cloud.google.com) project with the
+[Cloud Build API](http://console.cloud.google.com/apis/library/cloudbuild.googleapis.com) enabled.
+You can enable the API with the following `gcloud` command:
+
+```
+gcloud services enable cloudbuild.googleapis.com
+```
+
+## Configure Spinnaker to Listen for Google Cloud Build Pub/Sub Notifications
+
+Google Cloud Build sends [Build Notifications](https://cloud.google.com/cloud-build/docs/send-build-notifications)
+when the state of your build changes. You can create a pipeline trigger to invoke a pipeline based
+on the status of your build.
+
+1. Create a Subscription object for the `cloud-builds` topic in your project:
+
+    ```
+    PROJECT_ID=
+    SUBSCRIPTION_NAME=googleCloudBuilds
+
+    gcloud pubsub subscriptions create $SUBSCRIPTION_NAME --topic projects/$PROJECT_ID/topics/cloud-builds
+    ```
+
+2. Configure and deploy Halyard with your newly created subscription.
+
+    ```
+    hal config pubsub google subscription add $SUBSCRIPTION_NAME --project $PROJECT_ID --subscription-name $SUBSCRIPTION_NAME
+
+    hal config pubsub google enable
+
+    hal deploy apply
+    ```
+
+## Configure Your Pipeline Trigger
+
+1. In your Pipeline configuration, click the **Configuration** stage on the far left of the pipeline diagram.
+
+2. Click **Automated Triggers**.
+
+3. In the **Type** field, select `Pub/Sub`.
+
+4. In the **Pub/Sub System Type** field, select `google`.
+
+5. In the **Subscription Name** field, select your `$SUBSCRIPTION_NAME` value.
+
+6. In the **Attribute Constraints** field, enter `status` in the **Key**, and `SUCCESS` (all upper case) in the **Value** field.
```

Comment:
Perhaps add something like "This would trigger on **all** successful Cloud Build builds. To trigger only on builds for a particular repo, or producing a particular image, use **Payload Constraints**."?